TESTS/netsocket/README.md

```diff
@@ -1654,6 +1669,311 @@ through a dedicated socket
 Echo server returns data to both threads and received data matches to
 send data. The additional thread isn't stopped prematurely.
 
+Test cases for DNS class
+---------------------------
+
+### ASYNCHRONOUS_DNS
+
+**Description:**
+
+Verify basic functionality of asynchronous DNS. Call `NetworkInterface::gethostbyname_async()` with a valid host name and verify result.
```

Comment:
"Verify the basic functionality of asynchronous DNS. Call `NetworkInterface::gethostbyname_async()` with a valid host name, and verify the result."

Please replace this everywhere it repeats.